### ISSUE TITLE
Cannot define a normal `transactional` method

### ISSUE THREAD

**Issue by MaryamZi** [ballerina-platform/ballerina-lang#25016]
**Description:**
$title.

**Steps to reproduce:**
```ballerina
type Foo client object {
    remote transactional function baz() {  // no error
    }

    transactional function qux() { // error: .::zxa.bal:5:19: invalid token 'transactional'
    }
};
```

**Affected Versions:**
Current master (slp3-SNAPSHOT)

**Comment by SupunS** [comment]
Needs to be fixed along with #24221

**Comment by SupunS** [comment]
Fixed with #25078
